Makefile

```diff
@@ -6,16 +6,23 @@ COMMANDS = cli/enry
 MAKEFILE = Makefile.main
 CI_REPOSITORY = https://github.com/src-d/ci.git
 CI_FOLDER = .ci
-
-# If you need to build more than one dockerfile, you can do so like this:
-# DOCKERFILES = Dockerfile_filename1:repositoryname1 Dockerfile_filename2:repositoryname2 ...
-
 $(MAKEFILE):
 	@git clone --quiet $(CI_REPOSITORY) $(CI_FOLDER); \
 	cp $(CI_FOLDER)/$(MAKEFILE) .;
-
 -include $(MAKEFILE)
 
+# Docsrv: configure the languages whose api-doc can be auto generated
+LANGUAGES = go
```

Comment:
There are also java bindings, should it be reflected here?

Reply:
Java api was not defined as a requirement; it could be added as a feature request.